desktop/apps/gene/client.js

```diff
@@ -42,7 +42,7 @@ function setupGenePage () {
       .html(html)
       .addClass('is-fade-in')
   })
-  gene.fetchArtists('related')
+  gene.fetchArtists('related', { headers: { 'X-Request-Id': sd.REQUEST_ID } })
```

Comment:
It pains me to see the header duplicated... I know we discussed it a little, but could we wrap the various fetches (4 in this PR) with a helper that merges in the `headers: { 'X-Request-Id': ...` option? (My js-fu is not up to the challenge.)

Reply:
In this case, since it's on the client-side, we could add to [here](https://github.com/artsy/force/blob/master/desktop/lib/global_client_setup.coffee#L86) to augment all of these backbone/jquery requests. On the server-side however, there's not a much better option as one needs to explicitly tie the header to the `req.id`, so the best you could do is something like `gene.fetchArtists('related', { req })`.

Reply:
Ohh 👏 👏 I'll follow up with a PR to use those ajax settings on the client-side.